eo-maven-plugin/src/test/java/org/eolang/maven/it/SnippetTestCase.java

```diff
@@ -301,18 +304,42 @@ private static String jdkExecutable(final String name) {
      * @return Classpath.
      */
     private static String classpath() {
-        return String.format(
-            ".%s%s",
-            File.pathSeparatorChar,
-            System.getProperty(
-                "runtime.jar",
-                Paths.get(System.getProperty("user.home")).resolve(
+        final String runtime;
+        final String property = System.getProperty("runtime.jar");
+        if (SnippetTestCase.isNotRealPath(property)) {
+            runtime = Paths.get(System.getProperty("user.home"))
+                .resolve(
                     String.format(
                         ".m2/repository/org/eolang/eo-runtime/%s/eo-runtime-%1$s.jar",
                         "1.0-SNAPSHOT"
                     )
-                ).toString()
-            )
+                ).toString();
+        } else {
+            runtime = property;
+        }
+        return String.format(
+            ".%s%s",
+            File.pathSeparatorChar,
+            runtime
         );
     }
+
+    /**
+     * Checks if the path is real path.
+     * @param path String path to check.
+     * @return True if path is real.
+     */
+    private static boolean isNotRealPath(final String path) {
+        boolean res = false;
+        if (Objects.isNull(path) || path.isEmpty() || StringUtils.isBlank(path)) {
+            res = true;
+        } else {
+            try {
+                Paths.get(path);
+            } catch (final InvalidPathException ignore) {
```

Comment:
@volodya-lombrozo what is the point of swallowing here? If the path is wrong, we should inform the user. The path is provided by the user, so it's his mistakes. Otherwise, if we swallow, the user will be very confused: he provided the path, but we are reading from another path. "Why?" -- he will ask.